R/conflicts.R

```diff
@@ -18,71 +18,76 @@
 tidymodels_conflicts <- function() {
   envs <- purrr::set_names(search())
   objs <- invert(lapply(envs, ls_env))
-
+  
```

Comment:
Awesome!

Small thing, could you go back and make the changes to remove the changes that are purely indentation such as thing one

Reply:
Sure thing... my CMD + I habit is a bit too strong :)